src/components/App/App.js

```diff
@@ -0,0 +1,15 @@
+import Navigation from '../../layout/navigation/Sidebar';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import AddMotorcycle from '../motorcycleForm/addMotorcycle';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Navigation />
+      <AddMotorcycle />
+    </div>
+  );
+}
+
+export default App;
```

Comment:
The component `AddMotorcycle` should have a route path since it is should not be visible on every page like the `Navigation` component.